### ISSUE TITLE
[Question][Hw3] Possible errata in Problem 6?

### ISSUE THREAD

**Issue by kohs100** [prosyslab-classroom/cs524-program-analysis#177]
Name: Hyungseok Ko
![image](https://github.com/prosyslab-classroom/cs524-program-analysis/assets/20630805/b9a631ab-4251-4217-a67b-ee2e45a012e6)

![image](https://github.com/prosyslab-classroom/cs524-program-analysis/assets/20630805/e3c379c7-a3a2-4934-b62d-d87ae3f98fdb)

Memory를 Abstraction 할 때 Symbol 집합 X에는 Sharp이 안붙는게 맞는거같은데 오타인걸까요?

It seems there should not be sharp on symbol domain X in definition of galois connection on memory domain.

**Comment by Re-st** [comment]
Yes. Odd errata in fact $\mathbb{M}^\sharp=\mathbb{X}\to\mathbb{Z}^\sharp$ missed another opportunity of errata.

**Comment by kohs100** [comment]
Thanks!
